gpustack/cmd/start.py

```diff
@@ -372,6 +372,12 @@ def setup_start_cmd(subparsers: argparse._SubParsersAction):
         help="Port to expose worker metrics.",
         default=get_gpustack_env("WORKER_METRICS_PORT"),
     )
+    group.add_argument(
+        "--instance-health-check-interval",
+        type=int,
+        help="Interval in seconds to check model instance health.",
+        default=get_gpustack_env("INSTANCE_HEALTH_CHECK_INTERVAL"),
```

Comment:
This is too advanced. Define in `envs/__init__.py` instead of exposing in the cli for now.